components/dashboard/src/components/podkit/switch/Switch.tsx

```diff
@@ -51,23 +51,26 @@ export interface SwitchInputFieldProps extends React.ComponentPropsWithoutRef<ty
     id?: string;
     label: ReactNode;
     description?: ReactNode;
+    labelLayout?: "row" | "column";
 }
 
 export const SwitchInputField = React.forwardRef<React.ElementRef<typeof SwitchPrimitives.Root>, SwitchInputFieldProps>(
```

Comment:
I wish we do not have to change podkit components like this. We do not use such props anywhere else in podkit, and it makes the component significantly more complex for just this one-off use-case.

Can we move this code to the dashboard components and compose it there? 🙏

Reply:
The component it's just a few lines, I would agree it a really big component handling multiple cases already, duplicating the code feels unnecessary.

Reply:
We try to not introduce our own arbitrary props, because most of the time it is not necessary, and all standard ones are documented in Radix UI docs. It is also not a common use case we have, it seems we would only use this here, no?

Let's not worry about it too much now and focus on merging, though.